server/testdata/endpoints/18_couper.hcl

```diff
@@ -18,6 +18,28 @@ server "couper" {
       status = 204
     }
   }
+
+  api {
+    endpoint "/api-abcdef" {
+      proxy = "test"
+      response {
+        status = 204
+      }
+    }
+    endpoint "/api-reuse" {
+      proxy = "test"
+      response {
+        status = 204
+      }
+    }
+
+    endpoint "/api-default" {
+      proxy = "defaultName"
+      response {
```

Comment:
remove explicit `response` for "defaultName" proxies and expect `200`

Reply:
Done